src/Extensions/ServiceFabricGuest.Abstractions/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace Microsoft.Omex.Extensions.ServiceFabricGuest.Abstractions
+{
+	/// <summary>
+	/// Extension to add Service Fabric Client dependencies to IServiceCollection
+	/// </summary>
+	public static class ServiceCollectionExtensions
+	{
+		/// <summary>
+		/// Register types required Service Fabric comunication
+		/// </summary>
+		public static IServiceCollection AddServiceFabricClient(this IServiceCollection serviceCollection)
+		{
+			return serviceCollection.AddServiceFabricClient<ServiceFabricClientWrapper>();
+		}
+
+		/// <summary>
+		/// Register generic types required Service Fabric comunication
+		/// </summary>
+		public static IServiceCollection AddServiceFabricClient<TClinet>(this IServiceCollection serviceCollection)
```

Comment:
I don't think there is any need of having this method regular `AddServiceFabricClient` is enougth